frontend/libs/studio-components/src/components/StudioParagraph/StudioParagraph.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Paragraph, type ParagraphProps } from '@digdir/designsystemet-react';
+
+export type StudioParagraphProps = ParagraphProps;
+
+export const StudioParagraph = ({
+  children,
+  'data-size': dataSize = 'sm',
+  ...rest
+}: StudioParagraphProps) => {
+  return (
+    <Paragraph {...rest} data-size={dataSize}>
+      {children}
+    </Paragraph>
+  );
+};
```

Comment:
Vi bruker `function` til å deklarere komponenter. Husk returtype.
```suggestion
export function StudioParagraph({
  children,
  'data-size': dataSize = 'sm',
  ...rest
}: StudioParagraphProps): ReactElement {
  return (
    <Paragraph {...rest} data-size={dataSize}>
      {children}
    </Paragraph>
  );
}
```